tools/Customization/DevHome.Customization/CustomControls/DevDriveCardBody.xaml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.Customization.CustomControls.DevDriveCardBody"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+    <!--
+        Grid containing name of dev drive, state of dev drive in text and a list of the dev drive properties.
+    -->
+    <Grid Grid.ColumnSpan="3" Style="{StaticResource CardBodyContainerStyle}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+        <!-- 64x64 image for the dev drive. -->
+        <Viewbox HorizontalAlignment="Left"
+                Grid.Column="0">
+            <Grid>
+                <!-- Image for card body. -->
+                <FontIcon Glyph="&#xEDA2;" FontSize="25" FontFamily="{ThemeResource AmcFluentIcons}"/>
+            </Grid>
+        </Viewbox>
+        <TextBlock Grid.Column="1" Margin="0 5 0 0"
+                Style="{StaticResource CardBodyStackPanelTextBlockStyle}"
+                Text="{x:Bind DevDriveLabel, Mode=OneWay}"
+                IsTextSelectionEnabled="True"/>
+        <Grid Grid.RowSpan="3" Grid.Column="2" HorizontalAlignment="Stretch">

```

Comment:
You have `Grid.RowSpan` but the `grid` does not have any `RowDefinitions`

Reply:
Row definitions are in place.